Src/BootCamp.Chapter/Item.cs

```diff
@@ -15,12 +15,98 @@ public decimal GetPrice()
         }
 
         private float _weight;
+        public float GetWeight()
+        {
+            return _weight;
+        }
 
         public Item(string name, decimal price, float weight)
         {
             _name = name;
             _price = price;
             _weight = weight;
         }
+
+        public static Item[] Append(Item[] array, Item toAppend)
+        {
+            var newArray = new Item[1 + array.Length];
+            for (int i = 0; i < array.Length; i++)
+            {
+                newArray[i] = array[i];
+            }
+            newArray[^1] = toAppend;
+            return newArray;
+
+        }
```

Comment:
You should remove this completely. That's not of a good use case here. Also - while not being specified - the inventory should be of fixed size. I asked Kai for clarification and he told so.

Reply:
What size is that?

Reply:
That is up to you I guess. Think about a valid size for an inventory. I chose 100 in my implementation but that might be a bit big.